packages/data-stores/src/launchpad-navigator/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface UpdateLaunchpadNavigatorResponse {
+	current_checklist: string;
```

Comment:
Per discussion: this should be nullable.
```suggestion
	current_checklist: string | null;
```